### PR TITLE
Implementing Activity Stream of Dashboard

### PR DESCRIPTION
**Pull request by kotwani2883**
## Explain the feature/fix

Implementing Activity Stream of Spa-ship Dashboard (Only Application deployed to be shown for dashboard activity Stream)

## Does this PR introduce a breaking change
No

[screen-capture (6).webm](https://user-images.githubusercontent.com/56580582/211544121-ddd20865-8125-4638-98bc-de3d11748f07.webm)

@hybridx @akhilmhdh  Kindly review

### CODE REVIEW

**Comment by akhilmhdh** [comment]
@kotwani2883 kindly change the `Deployment Time: 1.134seconds` to a smaller format like `Deployment Time: 1.13s`

Just two point floating is required and s will suffice to notify seconds

**Comment by kotwani2883** [comment]
> @kotwani2883 kindly change the `Deployment Time: 1.134seconds` to a smaller format like `Deployment Time: 1.13s`
> 
> Just two point floating is required and s will suffice to notify seconds and kindly add some spacing between labels. It feels too congested. Also change the `tick-completed` label to compact also

Heyy @akhilmhdh this is according to the previous activity stream. That component is directly used here.Should I change that?

**Comment by akhilmhdh** [comment]
@kotwani2883 yah sure. I think the content has changed a bit. Yup please adjust it

**Comment by soumyadip007** [comment]
@akhilmhdh  I'll do it from the api.

**Comment by soumyadip007** [comment]
@akhilmhdh The changes will be done from the BE. We'll fix it from the API.

**Comment by akhilmhdh** [comment]
> @akhilmhdh The changes will be done from the BE. We'll fix it from the API.

@soumyadip007 its sending time zone right. Not text .. right

**Comment by soumyadip007** [comment]
Yes @akhilmhdh

**Review by hybridx — CHANGES_REQUESTED** [review]

File: packages/manager/src/views/DashboardPage/DashboardPage.tsx [hunk 1]
```diff
@@ -1,11 +1,13 @@
...
- </>
+export const DashboardPage = () => (
+  <div style={{ display: 'flex', flexDirection: 'row', height: '10%' }}>
+    <div style={{ width: '55%' }}>
+      <Analytics />
+    </div>
+    <div style={{ width: '45%' }}>
+      <ActivityStreamDashboard />
+    </div>
+  </div>
```
[the remark this answers is not in the document]
Comment (kotwani2883) [reply packages/manager/src/views/DashboardPage/DashboardPage.tsx:12]: Yes it does!

**Comment by kotwani2883** [comment]
Updating the branch with the master , created new PR -> https://github.com/spaship/spaship/pull/2099
hence closing this!

**Pull request closed without merging**